packages/table/src/table.tsx

```diff
@@ -546,7 +546,7 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
                     quadrantRef={this.quadrantRefHandlers[QuadrantType.MAIN].quadrant}
                     quadrantType={QuadrantType.MAIN}
                     renderBody={this.renderBody}
-                    renderColumnHeader={this.renderColumnHeader}
+                    renderColumnHeader={this.renderMainQuadrantColumnHeader}
```

Comment:
One of us is going to have merge conflicts as of Part 6 (#1390). Would it be easier to merge that first and then fix the issues here, or vice versa?